Sources/NIOSSL/SSLErrors.swift

```diff
@@ -58,6 +58,7 @@ public enum NIOSSLError: Error {
     case cannotFindPeerIP
     case readInInvalidTLSState
     case uncleanShutdown
+    case closeTimedOut
```

Comment:
this would be SemVer major, you'll need to create a new error type for that :|

Reply:
Agh -- removed!